src/app/components/common/TagResetButton.tsx

```
@@ -0,0 +1,26 @@
+  const searchParams = useSearchParams();
+  const hasTags = searchParams.get("tag");
+
+  // if (!hasTags) return null;
+
+  return (
+    <button
+      aria-label="초기화"
+      onClick={() => router.push("/")}
+      className="text-xs text-ne cursor-pointer transition-colors"
+      style={{ color: "var(--color-text-muted)" }}
+    >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)